apps/ejabberd/include/mod_muc_light.hrl

```diff
@@ -75,13 +75,17 @@
           raw_config = [] :: raw_config()
          }).
 
+-type op_config() :: #config{}.
```

Comment:
It's not that long :) IMO, the prettiest would be to move this type to `mod_muc_light` and name it `config`. Then you will use type like `mod_muc_light:config()`. I know this is even longer :)